packages/datadog-plugin-sharedb/src/index.js

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+const MessagesAwaitingResponse = new WeakMap();
```

Comment:
Any way to store the spans without relying on a global `WeakMap`? Since it's a weak map it shouldn't matter either way, but I tend to prefer solutions that avoid globals whenever possible.

Reply:
Unfortunately, no. This is also the suggested way to do this from the core maintainers of ShareDB.

Reply:
Any link to that recommendation that could be added in a comment for future reference?